<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageChecker</class>
 <widget class="QDialog" name="ImageChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check images</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelImageType">
       <property name="text">
        <string>Image type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxImageType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select image type to be checked</string>
       </property>
       <item>
        <property name="text">
         <string>Previews</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/camera.png</normaloff>:/data/img/camera.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Flyers</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/thumbnail.png</normaloff>:/data/img/thumbnail.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cabinets</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/arcadecabinet.png</normaloff>:/data/img/arcadecabinet.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Controllers</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/joystick.png</normaloff>:/data/img/joystick.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marquees</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/marquee.png</normaloff>:/data/img/marquee.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Titles</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/arcademode.png</normaloff>:/data/img/arcademode.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PCBs</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/circuit.png</normaloff>:/data/img/circuit.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Icons</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/icon.png</normaloff>:/data/img/icon.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelThreads">
       <property name="text">
        <string>Threads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxThreads">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of parallel threads used when checking images</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSelectSets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select sets in master list when selected in the found- or missing-lists below</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/ok.png</normaloff>:/data/img/ok.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="toolTip">
        <string>Current status / progress</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>Idle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonStartStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Start / stop checking images</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/refresh.png</normaloff>:/data/img/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRemoveObsolete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Remove obsolete image files</string>
       </property>
       <property name="icon">
        <iconset resource="qmc2.qrc">
         <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCheckResults">
      <attribute name="title">
       <string>Check results</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutCheckResults">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFound">
         <property name="text">
          <string>Found:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelMissing">
         <property name="text">
          <string>Missing:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelObsolete">
         <property name="text">
          <string>Obsolete:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="listWidgetFound">
         <property name="toolTip">
          <string>Sets for which at least one valid image was found</string>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="listWidgetMissing">
         <property name="toolTip">
          <string>Sets for which no valid images could be found</string>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QListWidget" name="listWidgetObsolete">
         <property name="toolTip">
          <string>Files which couldn't be matched to any set (and are thus obsolete)</string>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLog">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutLog">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEditLog">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxImageType</tabstop>
  <tabstop>spinBoxThreads</tabstop>
  <tabstop>toolButtonStartStop</tabstop>
  <tabstop>toolButtonRemoveObsolete</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidgetFound</tabstop>
  <tabstop>listWidgetMissing</tabstop>
  <tabstop>listWidgetObsolete</tabstop>
  <tabstop>plainTextEditLog</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
